packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -49,56 +47,11 @@ const AdvertiserPage = () => {
     // rating_average_decimal converts rating_average to 1 d.p number
     const rating_average_decimal = rating_average ? Number(rating_average).toFixed(1) : null;
     const joined_since = daysSince(created_time);
-    const error_message = () => {
-        return !!advertiser_page_store.is_counterparty_advertiser_blocked && !is_my_advert
-            ? localize("Unblocking wasn't possible as {{name}} is not using Deriv P2P anymore.", {
-                  name: advertiser_page_store.advertiser_details_name,
-              })
-            : localize("Blocking wasn't possible as {{name}} is not using Deriv P2P anymore.", {
-                  name: advertiser_page_store.advertiser_details_name,
-              });
-    };
 
     React.useEffect(() => {
         advertiser_page_store.onMount();
         advertiser_page_store.setIsDropdownMenuVisible(false);
-
-        reaction(
```

Comment:
I've removed the reaction here and moved it to the general_store since the block_user.tsx's reaction is similar, i've just adjusted the logic to it @nada-deriv

Reply:
noted. will make the corresponding change in mine as well when fixing the conflicts.